src/server/game/Spells/SpellInfoCorrections.cpp

```diff
@@ -4421,6 +4421,16 @@ void SpellMgr::LoadSpellInfoCorrections()
         spellInfo->Effects[EFFECT_0].MiscValueB = 64;
     });
 
+    // Eye of Kilrogg Passive (DND)
+    ApplySpellFix({ 58083 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_2].Effect = SPELL_EFFECT_APPLY_AURA;
+        spellInfo->Effects[EFFECT_2].BasePoints = 49;
+        spellInfo->Effects[EFFECT_2].DieSides = 1;
+        spellInfo->Effects[EFFECT_2].TargetA = SpellImplicitTargetInfo(TARGET_UNIT_CASTER);
+        spellInfo->Effects[EFFECT_2].ApplyAuraName = SPELL_AURA_MOD_INCREASE_SPEED;
+    });
```

Comment:
Why

Reply:
@Kitzunu, the Spell Glyph of Kilrogg  does not work, as it also serves as a basis in the script